### DISCUSSION TITLE
tensor flow issue

### DISCUSSION THREAD

**Discussion by Aqsa48** [AammarTufail/python_ka_chilla2023#94] [Q&A]
![image](https://user-images.githubusercontent.com/21342218/215557664-654b9112-0b0a-4001-a04c-b51b5805613e.png)
![image](https://user-images.githubusercontent.com/21342218/215557715-e82985da-7140-4af5-b6db-d66021543067.png)

why library is giving this issue?

**Comment by Sahibzada18** [comment]
mean_square_error is not recognized by TensorFlow.Try importing the loss function from library "from tensorflow.keras.losses import mean_squared_error"

**Reply by Aqsa48** [reply]
![image](https://user-images.githubusercontent.com/21342218/215793018-abae706f-8bbf-4ab9-bc5b-f797cabc0014.png)

providing error like this.

**Reply by Sahibzada18** [reply]
![tf_mean_squared_error](https://user-images.githubusercontent.com/38686289/215799199-0178538b-1720-4aa2-8a69-6cabed1cf79b.png)
 Try this :) and then let me know

**Reply by Sahibzada18** [reply]
And to resolve that yellow line below tensorflow set "reportMissingImports" setting to "ignore" in your configuration @Aqsa48

**Reply by Aqsa48** [reply]
@Sahibzada18 

Still same error.

**Reply by Sahibzada18** [reply]
try restarting your VS code

**Reply by Sahibzada18** [reply]
try clearing the VS Code cache by deleting the .vscode folder in your project directory. Hope it Work out for you

**Comment by Muhammad-Ali-Butt** [comment]
i was getting different errors what i did was delete the entire environment from conda and then create new env and installed tensorflow then installed other libraries except numpy as it was alr installed with tensorflow. all my errors were resolved after deleting the previous env and creating a new one.

**Reply by Aqsa48** [reply]
it didn't work.

**Reply by Muhammad-Ali-Butt** [reply]
nevermind it worked for me i basically opened conda then deleted current env and made new one with python version 3.9(conda create --name tf_env python=3.9) then installed tensorflow without specifying any version i.e. i initially downloaded tensorflow like this: **pip install tensorflow<11**  so now i installed it simply as **pip install tensorflow** . after this pip installed seaborn, plotly, openpyxl, pandas, matplotlib plus scikit-learn and thats it. didnt get any errors after i did this and later in vscode using this very environment i pip installed opencv as well and everything has been working like a charm ever since
